packages/core/src/services/createSignedURLService.ts

```diff
@@ -6,27 +6,26 @@ import {
 } from '@unchainedshop/core-files';
 
 export type CreateSignedURLService = (
-  params: { directoryName: string; fileName: string; meta?: any },
-  unchainedAPI: { modules: { files: FilesModule } },
+  params: { directoryName: string; fileName: string; meta?: any; isPrivate?: boolean },
+  unchainedAPI: { userId?: string; modules: { files: FilesModule } },
 ) => Promise<SignedFileUpload>;
 
 export const createSignedURLService: CreateSignedURLService = async (
-  { directoryName, fileName, meta },
+  { directoryName, fileName, meta, isPrivate },
   unchainedContext,
 ) => {
   const {
     modules: { files },
   } = unchainedContext;
   const fileUploadAdapter = getFileAdapter();
-  const preparedFileData = await fileUploadAdapter.createSignedURL(
-    directoryName,
-    fileName,
-    unchainedContext,
-  );
-  const fileData = getFileFromFileData(preparedFileData, meta);
+  const preparedFileData = await fileUploadAdapter.createSignedURL(directoryName, fileName);
```

Comment:
Maybe there is no need to adjust this service at all, isPrivate and userId could be in meta, userId is already in meta for avatars for ex..